plugins/mongodb/mongo_collection_

```diff
@@ -29,8 +29,8 @@
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
 # SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-#%# family=auto
-#%# capabilities=suggest autoconf
+# %# family=auto
+# %# capabilities=suggest autoconf
```

Comment:
`#%#` is a special marker. In this case you would need to use `# #%#` in order to satisfy the python style requirement, as well as munin's marker format.

Reply:
here we are